sql/executor.go

```diff
@@ -328,6 +336,33 @@ func (cw *logChanWriter) Close() {
 	}
 }
 
+func trainWithoutValidation(tr *extendedSelect, slct string, db *DB, cwd string, wr *PipeWriter, modelDir string) error {
```

Comment:
This function is quite the same as `train()`, curious why we need to add this function now?

Reply:
I filed 4 related PRs to implement the whole design. And this function used for a placeholder appears too early in this PR.
I'm taking it to the next PR.